recipe/activate.sh

```diff
@@ -8,6 +8,7 @@ export RUSTUP_HOME=${CARGO_HOME}/rustup
 
 export CARGO_TARGET_@rust_arch_env_build@_LINKER=${CC_FOR_BUILD:-${CONDA_PREFIX}/bin/@rust_default_cc_build@}
 export CARGO_TARGET_@rust_arch_env@_LINKER=${CC:-${CONDA_PREFIX}/bin/@rust_default_cc@}
+export CARGO_TARGET_@rust_arch_env_build@_RUSTFLAGS="-C link-arg=-Wl,-rpath,${BUILD_PREFIX:-${CONDA_PREFIX}}/lib"
```

Comment:
If you are not cross compiling, I don't see a reason why we need this.

Reply:
Ok I wasn't sure about this.

So I can add this only for packages that are cross compiling.